runtime/doc/lua.txt

```diff
@@ -2513,6 +2513,180 @@ vim.loader.reset({path})                                  *vim.loader.reset()*
       • {path}  (`string?`) path to reset
 
 
+==============================================================================
+Lua module: vim.pack                                                *vim.pack*
+
+WORK IN PROGRESS built-in plugin manager! Early testing of existing features
+is appreciated, but expect breaking changes without notice.
+
+Manages plugins only in a dedicated |packages| *vim.pack-directory*:
+`$XDG_DATA_HOME/nvim/site/pack/core/opt`. Plugin's subdirectory name matches
+plugin's name in specification. It is assumed that all plugins in the
+directory are managed exclusively by `vim.pack`.
+
+Uses Git to manage plugins and requires present `git` executable of at least
+version 2.36. Target plugins should be Git repositories with versions as named
+tags following semver convention `v<major>.<minor>.<patch>`.
+
+Example workflows ~
+
+Basic install and management:
+• Add |vim.pack.add()| call(s) to 'init.lua': >lua
+
+    vim.pack.add({
+      -- Install as "plugin1"/"plugin2" and use greatest available version
+      'https://github.com/user/plugin1',
+      { source = 'https://github.com/user/plugin2' },
+
+      -- Specify plugin's name
+      { source = 'https://github.com/user/generic-name', name = 'plugin3' },
+
+      -- Specify version to follow during install and update
+      {
+        source = 'https://github.com/user/plugin4',
+        version = vim.version.range('1.0'), -- Greatest version matching range
```

Comment:
I'm not sure what "Greatest version matching range" means. Maybe call this the "version constraint" instead (with a link to the docs).

I had to read the `vim.version.range` docs to understand exactly what `vim.version.range('1.0')` means. This matches `1.0.0` and `1.0.1`, but not `1.1`, for example.

```suggestion
        version = vim.version.range('1.0'), -- Version constraint. See |vim.version.range()|
```

Reply:
> I'm not sure what "Greatest version matching range" means. Maybe call this the "version constraint" instead (with a link to the docs).

In `vim.version` terminology it is "last", which I don't find correct/intuitive. It is the largest/biggest/greatest available version that matches any constraint specified via `vim.version.range()`. In this particular case it is indeed larger than or equal `1.0.0` but strictly less than `1.1.0`.

Yes, linking to `vim.version.range()` makes sense.